commands/cursor.go

```diff
@@ -71,6 +71,12 @@ func (cmd *Cursor) Parse() error {
 	case "pgdn", "pagedn", "pagedown":
 		_, y := songlistWidget.Size()
 		cmd.relative = y
+	case "halfpageup":
+		_, y := songlistWidget.Size()
+		cmd.relative = -y / 2
+	case "halfpagedown":
```

Comment:
Suggest also adding `halfpgup`, `halfpagedn` and `halfpgdn`, to match the existing dictionary?